.github/workflows/build_and_deploy.yaml

```diff
@@ -0,0 +1,46 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches:
+      - main
+      - feat/ci-cd
```

Comment:
Is this branch still needed? this tells that only build when changes are made either on main branch or feat/ci-cd.

Reply:
This is for future edge case implementation